Package.swift

```diff
@@ -41,18 +41,18 @@ let package = Package(
                 "SwiftyBeaver",
                 "Gloss",
                 "PIALibraryUtilObjC",
-                "SwiftEntryKit",
-                "PopupDialog",
+                .product(name: "PopupDialog", package: "PopupDialog", condition: .when(platforms: [.iOS])),
+                .product(name: "SwiftEntryKit", package: "SwiftEntryKit", condition: .when(platforms: [.iOS])),
                 .product(name: "Lottie", package: "lottie-ios"),
                 .product(name: "Reachability", package: "Reachability.swift"),
                 .product(name: "PIAKPI", package: "mobile-ios-releases-kpi"),
                 .product(name: "PIACSI", package: "mobile-ios-releases-csi"),
                 .product(name: "PIARegions", package: "mobile-ios-releases-regions"),
                 .product(name: "PIAAccount", package: "mobile-ios-releases-account"),
                 .product(name: "PIAWireguard", package: "mobile-ios-wireguard"),
-                .product(name: "TunnelKit", package: "mobile-ios-openvpn"),
-                .product(name: "TunnelKitOpenVPN", package: "mobile-ios-openvpn"),
-                .product(name: "TunnelKitOpenVPNAppExtension", package: "mobile-ios-openvpn"),
+                .product(name: "TunnelKit", package: "mobile-ios-openvpn", condition: .when(platforms: [.iOS])),
+                .product(name: "TunnelKitOpenVPN", package: "mobile-ios-openvpn", condition: TargetDependencyCondition.when(platforms: [.iOS])),
+                .product(name: "TunnelKitOpenVPNAppExtension", package: "mobile-ios-openvpn", condition: TargetDependencyCondition.when(platforms: [.iOS]))
```

Comment:
I just noticed the condition. Does it mean we are not planning to support ovpn on tvOS?

Reply:
Good catch! Not for now. There were still some issues with OpenSSL on tvOS that we will address on the future. For now let's make tvOS work and connect with IKEV2, then we can fix OpenVPN.
Thank you @kp-juan-docal!